services/api/src/fixtures.js

```diff
@@ -1,15 +1,28 @@
-const { User, Product, Shop, Upload, Category } = require('./models');
+const { User, Role, Product, Shop, Upload, Category } = require('./models');
 const config = require('@bedrockio/config');
 const { storeUploadedFile } = require('./utils/uploads');
 const { logger } = require('./utils/logging');
+const { createMaxPermissions } = require('./utils/permissions');
 
 const adminConfig = {
   name: config.get('ADMIN_NAME'),
   email: config.get('ADMIN_EMAIL'),
   password: config.get('ADMIN_PASSWORD'),
-  roles: ['admin'],
 };
 
+const roles = [
+  {
+    name: 'Super Admin',
```

Comment:
I don't know if this will eventually appear anywhere but I'm fine with just "Admin" and "Owner" ... "Admin" has a sort of technical dangerous feel that I feel people understand is meant to be used with caution, also pretty common pattern... "super" just makes me think wait is there a "normal admin"?

just an opinion though

Reply:
guess we're going with "Super Admin" in parallel with multi-tenancy so this makes more sense now

Reply:
guess we're going with "Super Admin" in parallel with multi tenancy so this makes sense